contracts/core/interfaces/ICoreAccounting.sol

```diff
@@ -0,0 +1,64 @@
+/*
+    Copyright 2018 Set Labs Inc.
+
```

Comment:
Instead of copying these contracts, I think we want to import it like we could `open-zeppelin-solidity` - so please remove these